src/Models/Nonhydrostatic3DModels/equations_moisture.jl

```diff
@@ -55,12 +56,12 @@
        Geometry.Covariant123Vector(interp_f2c(w))
 
     # hyperdiffusion
-    dρq_tot = 0.0 .* dρq_tot
+    #dρq_tot = FT(0) .* dρq_tot
     χq = @. hwdiv(hgrad(ρq_tot / ρ))
     Spaces.weighted_dss!(dρq_tot)
     @. dρq_tot = -κ₄ * hwdiv(ρ * hgrad(χq))
```

Comment:
If the working fluid is moist air then the diffusive / hyperdiffusive moisture terms should appear in the tendency equation for \rho as well.

Reply:
Not sure, I understand :).

Reply:
For now, we are going to ignore the effect :)